modules/installation-configuration-parameters.adoc

```diff
@@ -1816,6 +1816,16 @@ at least two zones.
 |Enables accelerated networking. Accelerated networking enables single root I/O virtualization (SR-IOV) to a VM, improving its networking performance.
 |`Accelerated` or `Basic`. If instance type of control plane and compute machines support `Accelerated` networking, by default, the installer enables `Accelerated` networking, otherwise the default networking type is `Basic`.
 
+|operatorPublishingStrategy:
```

Comment:
`operatorPublishingStrategy` requires `publish` setting to "Mixed", better to mention it here.

Reply:
Thanks for that information. I have updated the PR

Reply:
Seems that below description is still not accurate. Whether `operatorPublishingStrategy.apiserver `or `operatorPublishingStrategy.ingress` set to `Internal` or `External`, parameter `publish` must be set to 'Mixed' if `operatorPublishingStrategy` is present in install-config.yaml.
```
 If you set this parameter to `Internal`, you must set the `publish` parameter to `Mixed`.
```

Reply:
OK, would it be accurate to instead say: 
```
If you set this parameter, you must set the `publish` parameter to `Mixed`.
```
?

Reply:
okay for me